doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -169,11 +176,11 @@ Traceability to Reference Model.
 | `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. |
 | `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. |
 | `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. |
-| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with ISO 27001 and adhere to the V1.1 NIST Cybersecurity Framework. |
-| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with ISO27001. |
+| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. |
```

Comment:
Do we need to keep ntw01~03 as a must (instead of should) ? Isn't **must** here a very hard requirement from CNTT perspective ? @pgoyal01  What is your view ?

Reply:
@msidana IMHO the access control requirements need to be a "must" but will let others chime in.

Reply:
@iangardner22 @CsatariGergely What are your views on this ?

Reply:
If we are intending to use kernel-based Neutron OVS with security groups for non-DPDK workloads then we can only meet some of these requirements (i.e. no logging).  However the word "underlay" suggests we are talking outside of OpenStack -  can someone clarify this?

If we are using OVS-DPDK and provider VLANs then we'd need a device that can implement these controls such as a firewall or SDN solution. 

I think MUST is OK - I just think we need to clarify "underlay" to make sure we all agree.